### PR TITLE
Add projects - IDE and Prometheus projects. 

### PR DESCRIPTION
**Pull request by CrazyByDefault**
Added projects, and basic descriptions. 

### Need help figuring out the milestones for them. 

@Gouthamve look at the Prometheus projects once and temme if they're alright.

### CODE REVIEW

File: data/projects.js [hunk 1]
```diff
@@ -46,5 +46,68 @@ projects = [
...
+	},
+	{
+		"name" : "Prometheus - Building a full integration testing environment",
+		"desc" : "Develop a full integration testing environment for Prometheus",
+		"requirements" : [
+				"Need to work >5hrs a day."
+		],
+		"difficulty" : "high",
+		"issues" : [
+			"https://github.com/prometheus/prometheus/issues/2497"
```
Comment (gouthamve) [inline data/projects.js:88]: Don't show up in UI.

File: data/projects.js [hunk 2]
```diff
@@ -11,14 +11,14 @@ projects = [
 		"difficulty" : "medium",
 		"issues" : [
 		],
-		"mentors" : ["Siddhartha1234"],
+		"mentors" : ["Someone"],
```
Comment (gouthamve) [inline data/projects.js:14]: cc @ayushpateria Want to do it?

**Review by gouthamve — APPROVED** [review]
LGTM besides the small nits.

File: data/projects.js (hunk 1 above)
Comment (CrazyByDefault) [reply data/projects.js:88]: They do...?

image

File: data/projects.js (hunk 2 above)
Comment (ayushpateria) [reply data/projects.js:14]: Not really interested in front end.

**Pull request merged**